advanced-achievements-plugin/src/main/java/com/hm/achievement/listener/statistics/AbstractRateLimitedListener.java

```diff
@@ -119,7 +124,7 @@ private boolean isInCooldownPeriod(Player player, int slotNumber) {
 	 */
 	private void displayActionBarMessage(Player player, long timeToWait) {
 		String timeWithOneDecimal = String.format("%.1f", (double) timeToWait / 1000);
-		String message = "&o" + StringUtils.replaceOnce(langStatisticCooldown, "TIME", timeWithOneDecimal);
+		String message = "\u00A7o" + StringUtils.replaceOnce(langStatisticCooldown, "TIME", timeWithOneDecimal);
```

Comment:
This is the formatting change.

Rest of changes in this file are just from when I saved it and auto formatting took over